player/configfiles.c

```diff
@@ -313,7 +320,7 @@ void mp_write_watch_later_conf(struct MPContext *mpctx)
         goto exit;
     }
 
-    mp_mk_config_dir(mpctx->global, MP_WATCH_LATER_CONF);
+    mp_mk_config_dir(mpctx->global, mpctx->cached_watch_later_configdir);
```

Comment:
It looks like `mp_get_playback_resume_config_filename` is called by `mp_load_playback_resume`, which itself should be called before `cached_watch_later_configdir` is required...I think.

Does that sound right or am I missing something?